src/main/java/gov/cabinetofice/gapuserservice/web/LoginController.java

```diff
@@ -11,37 +13,58 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.servlet.view.RedirectView;
 
+import java.util.function.Supplier;
+
 @RequiredArgsConstructor
 @Controller
 public class LoginController {
 
     private final ThirdPartyAuthProviderProperties authenticationProvider;
-    private final ThirdPartyJwtService colaJwtService;
-    private static final String REDIRECT_URL_COOKIE = "redirectUrl";
-    private static final String COLA_COOKIE_NAME = "find-grants-test";
+    private final ColaJwtServiceImpl thirdPartyJwtService;
```

Comment:
Would like us to look at the interface that these classes both implement and work out whether we could make more of the methods public as the only reason we import two implementations here rather than two instances of the interface is because we didn't have enough public access to methods.